ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -992,7 +993,17 @@ vlan_internal_allocation_policy:
 ```yaml
 ip_dhcp_relay:
   information_option: < true | false >
+```
+
+### IP ICMP
 
+```yaml
+ip_icmp:
+  rate_limit_unreachable: < true | false >
+  redirect: <true | false >
+  source_interface:
```

Comment:
The CLI supports multiple source interfaces. One per VRF. The data model should allow this.